components/HeroUnit.js

```diff
@@ -75,11 +78,28 @@ function HeroUnitFooter () {
       <div className='nav'>
         <NavigationBar />
       </div>
-      <DownArrowIcon />
+      <Link to='tickets' spy smooth duration={500}>
+        <div className='link'>
+          <DownArrowIcon />
+        </div>
+      </Link>
+
       <style jsx>{`
         .nav {
           margin-bottom: 30px;
         }
+        .link {
+          cursor: pointer;
+          display: inline-block;
+          -webkit-transition: transform 0.25 ease-in-out;
```

Comment:
Since [most browsers support CSS transitions now](http://caniuse.com/#feat=css-transitions), I think we no longer need to use vendor prefixes.